src/cascadia/TerminalSettingsEditor/ProfileViewModel.cpp

```diff
@@ -768,72 +763,50 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         }
     }
 
-    // Method Description:
-    // - Check if the bell sounds exist on disk. Mark any that don't exist
-    //   so that they show the appropriate UI
-    safe_void_coroutine ProfileViewModel::_CheckBellSoundsExistence()
+    BellSoundViewModel::BellSoundViewModel(const Model::IMediaResource& resource) :
+        _resource{ resource }
     {
-        co_await winrt::resume_background();
-        std::vector<Editor::BellSoundViewModel> markedSounds;
-        for (auto&& sound : _CurrentBellSounds)
-        {
-            if (!std::filesystem::exists(std::wstring_view{ sound.Path() }))
-            {
-                markedSounds.push_back(sound);
-            }
-        }
-
-        co_await winrt::resume_foreground(_dispatcher);

```

Comment:
we don't need this any more because the settings model did it and populated them all with `.Ok()`